### PR TITLE
MSUIF Alert box component

### PR DESCRIPTION
**Pull request by damianmuti**
Guys, take a look at the finished component. Besides using variables for almost every aspect, the component provides the posibility to have different in/out animations as well as different types of alert boxes.

### CODE REVIEW

File: assets/templates/index.html [hunk 1]
```diff
@@ -34,10 +34,15 @@
   <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
   <meta name="msapplication-TileColor" content=""><!-- Add brand color -->
   <meta name="msapplication-TileImage" content="/mstile-144x144.png">
+  <style>
+    .choto:checked {
```
Comment (damianmuti) [inline assets/templates/index.html]: Haha! My bad. Removing this line in the next commit.

**Comment by arzafran** [comment]
Great work pal! :100:

**Comment by nazarenooviedo** [comment]
:+1:

File: assets/styles/modules/_alerts.scss [hunk 2]
```diff
@@ -1,13 +1,158 @@
...
+  }
+
+  to {
+    transform: translateX(-#{$alert-toast-max-width + $alert-toast-margin});
+  }
+}
+
+.c-alert {
+  position: fixed;
+  z-index: 1000; // This value can go as high as needed depending on the layout
```
Comment (fcoronelmakingsense) [inline assets/styles/modules/_alerts.scss]: Maybe value in a variable?

File: assets/styles/modules/_alerts.scss [hunk 3]
```diff
@@ -1,13 +1,158 @@
...
+  padding: toem(20px);
+
+  // Toast type
+  &--toast {
+    max-width: $alert-toast-max-width;
+    margin: $alert-toast-margin;
+    border-radius: toem(7px);
```
Comment (fcoronelmakingsense) [inline assets/styles/modules/_alerts.scss]: Maybe value in a variable for custom value?

**Comment by fcoronelmakingsense** [comment]
Very nice work!! :clap: :clap:

File: assets/styles/modules/_alerts.scss (hunk 3 above)
Comment (damianmuti) [inline assets/styles/modules/_alerts.scss]: Yep, I like the idea. I'll update this PR. Thanks, Fede!

File: assets/styles/modules/_alerts.scss (hunk 2 above)
Comment (damianmuti) [inline assets/styles/modules/_alerts.scss]: Will do. Thanks for the feedback, Fede!

**Comment by lcalvino** [comment]
great great great!

**Pull request merged**